md/configure-client-of-bonita-bpm-engine.md

```diff
@@ -32,7 +32,7 @@ APITypeManager.setAPITypeAndParams(ApiAccessType.HTTP, parameters)
 ```
 
 ### Configure Client using System properties
-You can set the system property `org.bonitasoft.engine.api-type` to `HTTP`, `EJB3` or `LOCAL`.
+You can set the system property `org.bonitasoft.engine.api-type` to `HTTP` or `LOCAL`.
```

Comment:
:grey_exclamation: Personally, I would have removed the EJB support in another PR

Reply:
Since they are tied to Wildfly, and there was not really a lot of them, figured I might as well do it here.

Reply:
ok, fine